src/sql/workbench/services/accountManagement/browser/media/accountDialog.css

```diff
@@ -12,6 +12,15 @@
 	padding-bottom: 15px;
 }
 
+.loading-provider-view {
```

Comment:
Is there a class on the overall dialog container you can use? It would be good to scope these down to that as well